### DISCUSSION TITLE
Logica specification

### DISCUSSION THREAD

**Discussion by saintserge** [EvgSkv/logica#40] [General]
Hi,
Thanks for awesome project. 
Will you have language specifications so I can implement Logica in other programming languages?
Or the way for now it is pass the official tests?

Thanks.

**Comment by EvgSkv** [comment]
Hello,
thanks for support!
We definitely need a proper language specification, but we don't have it yet.
I do plan to check in schema for the syntax tree. It should be helpful for the implementation.
Making sure tests pass is always a good idea, but the tests were not designed to ensure full language correctness.
If you would like to work on writing the spec, I'd be happy to answer any questions you get on the way :-)
Thank you!
